client/app/lib/helpers/url-helpers.js

```diff
@@ -41,5 +41,26 @@ function getSurveyId() {
   return match && match[1];
 }
 
+/**
+ * Get the assignment id from URL.
+ *
+ * return {number}
+ */
+function getAssessmentId() {
+  const match = window.location.pathname.match(/^\/courses\/\d+\/assessments\/(\d+)/);
```

Comment:
Why is this duplicated in here and in `client/app/api/course/assessment/Base.js`?